source/agora/consensus/EnrollmentManager.d

```diff
@@ -112,13 +118,6 @@ public class EnrollmentManager
             break;
         }
 
-        // load preimages
-        results = this.db.execute("SELECT val FROM node_enroll_data " ~
-            "WHERE key = ?", "preimages");
-
-        if (!results.empty)
-            this.preimages = results.oneValue!(ubyte[]).deserializeFull!(Hash[]);
```

Comment:
I found that we don't need to store them. It can be created or restored in the two places where the enrollment manager is creating pre-images and getting a preimage for revealing the one.

Reply:
That makes sense.